backend/src/migrations/20240333221317_add_user_friends_table.ts

```diff
@@ -0,0 +1,46 @@
+import { type Knex } from 'knex';
+
+import { DatabaseTableName } from '~/common/database/database.js';
+
```

Comment:
Don't use other dependency in your migrations. Fix this in achievements too.

Reply:
Thank you for reviewing. I have already fixed it.